apps/store-app/src/app/store.config.ts

```diff
@@ -1,4 +1,4 @@
-import { Entity, RESTResource, FundamentalStoreConfig } from '@fundamental-ngx/store';
+import { Entity, RESTResource, FundamentalStoreConfig, BaseEntity, ChainingPolicy } from '@fundamental-ngx/store';
 
```

Comment:
Do you want handle example for req. chaning in the next PR?

Reply:
I added example for request chaining